### DISCUSSION TITLE
Medusa Twilio SMS Plugin

### DISCUSSION THREAD

**Discussion by RaviUllagaddi** [medusajs/medusa#3174] [Q&A]
We installed Medusa Twilio SMS Plugin and also configured with whatsapp number. trying to trigger the send sms functionality but getting an error and not getting any messages in whatsapp.

is medusa twilio sendsms plugin supports for whatsapp ?

**Comment by srindom** [comment]
I am not sure if Twilio supports WhatsApp, you'd have to double-check. If you think the error you are seeing is a bug in the plugin please open an issue instead and include reproduction steps :)
